src/app.rs

```diff
@@ -698,6 +698,58 @@ where
         self
     }
 
+    /// Register the default asset store with the loader logic of the Application.
```

Comment:
Please could you use "Registers .."? It's *really* pedantic, but the [RFC doc summary sentence](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#summary-sentence) is stuck in my head.